### PR TITLE
Quick add a Work Item Type(almighty/almighty-core#99 UI Issue 3).

### PR DESCRIPTION
**Pull request by nimishamukherjee**
When viewing a list of work items, the top row will be used to quickly add a work item.
When the item name is entered the add button (+) gets enabled. After adding, the list of work items are displayed in reversed order so that the item that was added is displayed at the top.

---
This change is [Reviewable](https://reviewable.io/reviews/almighty/almighty-ui/23)

### CODE REVIEW

**Comment by mindreeper2420** [comment]
@nimishamukherjee @joshuawilson FYI, this does not match the design specs from the UXD Team. It will need to be altered to match the proper specs. @mgranfie

**Comment by nimishamukherjee** [comment]
@mindreeper2420 we have used the patternfly form component here. This is a temporary UI. Once the final quick add flow and screen designs are available, we will update this screen,

File: src/app/login/login.service.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { LoginItem } from './login-item';
 @Injectable()
 export class LoginService {
   // private githubUrl = 'app/loginStatus';
-  private githubUrl = 'http://localhost:8081/api/login/authorize ';  // URL to web api
+  private githubUrl = 'http://localhost:8080/api/login/authorize ';  // URL to web api
```
Comment (joshuawilson) [inline src/app/login/login.service.ts]: Why did you change this? It is set to 8081 so that it doesn't conflict with the backend when running on localhost.

**Comment by joshuawilson** [comment]
What is the second commit for? It appears to deal with the Login and the Details pages.

**Comment by joshuawilson** [comment]
@nimishamukherjee at the moment we need something that is better then we had. This commit fills in the gap. We know it is a throw away design.

File: src/app/work-item/work-item-detail/work-item-detail.component.html [hunk 2]
```diff
@@ -1,31 +1,61 @@
...
+  <br />
+  <div class="form-horizontal">
+    <div class="form-group">
+      <label class="col-sm-1 control-label">ID</label>
+      <div class="col-sm-5">
+        {{workItem.id}}
+      </div>
+    </div>
+    <div class="form-group">
+      <label class="col-sm-1 control-label" for="name">Name</label>
```
Comment (joshuawilson) [inline src/app/work-item/work-item-detail/work-item-detail.component.html]: The use of 'for' does not have anything to refer to. You need to add an 'id' or 'name' attribute to the div the label is 'for'.

**Comment by nimishamukherjee** [comment]
@joshuawilson removed the second commit.
